install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -29,6 +29,10 @@ spec:
           - "discovery"
           - "--admission-service"
           - "istio-pilot"
+{{- if .Values.global.multiclusterEnabled }}
+          - "--clusterRegistriesDir"
```

Comment:
Do you mind to add TODO here to go back and change it after secret refactor PR gets merged.

Reply:
it will be top of mind in my head and I'll take care of whatever needs changing here.

Cheers
-steve